ui/src/alerting/components/builder/CheckConditionsCard.tsx

```diff
@@ -26,60 +23,35 @@ interface StateProps {
   check: Partial<Check>
 }
 
-interface DispatchProps {
-  changeCheckType: typeof changeCheckType
-}
+type Props = StateProps
+
+const CheckConditionsCard: FC<Props> = ({check}) => {
+  let cardTitle: string
+  let conditionsComponent: JSX.Element
 
-type Props = DispatchProps & StateProps
+  if (check.type === 'deadman') {
+    cardTitle = 'Deadman'
```

Comment:
I wonder if this should be Deadman Conditions?

Reply:
Ahh like split it into two components entirely?

Reply:
I meant should the cardTitle be "Deadman Conditions"